### DISCUSSION TITLE
İbax Formuna Özel Rütbe Ve Stickerlar!

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#460] [Ideas]
### Clear and concise description of the problem

İbax formunu aktif kullanan üyelerin profil resimleri, aktiflik ve tecrübelik durumuna göre belirlenmeli ve bu resimler tamamen İBAX tarafından atanmalıdır

### Suggested solution

Aktif ve yarar sağlayan üyelere özel rütbe esasına göre profil resimleri( rütbeye göre kilit açma) olmalıdır.

### Alternative

1-100 seviye belirlenmeli ve belli bir tecrübe puanında seviye atlanılmalı. Ardından 1-100 adet arası profil resmi belirlenmelidir

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment]
The IBAX team welcomes everyone to participate in our discussions, and we prefer developers who can help the project develop and grow more quickly, so the forum is hosted on Github for easier code interaction.

You can download GitHub mobile version.
